analysis/q_gdn_ddcap/phase_c.py

```
@@ -0,0 +1,420 @@
+            m["dd_ci"] = ci(dds)
+            m["net_ci"] = ci(nets)
+            results[label][cohort_label] = m
+            boots[label][cohort_label] = {"pf": pfs, "dd_pct": dds, "net": nets}
+
+    gate = evaluate_gate(results, boots)
+
+    # ----- Console summary -----
+    print("=" * 78)
+    print("Q-GDN-DDcap Phase C — H1/H2 panel split with block bootstrap (1000 iter)")
+    print("=" * 78)
+    print(f"Static-equity convention: per-trade Net P&L % * $200K / 100")
```

Ruff (0.15.13)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)